src/topology/algebra/ordered.lean

```diff
@@ -1944,3 +1944,81 @@ lemma tendsto_of_monotone {ι α : Type*} [preorder ι] [topological_space α]
   tendsto f at_top at_top ∨ (∃ l, tendsto f at_top (𝓝 l)) :=
 if H : bdd_above (range f) then or.inr ⟨_, tendsto_at_top_csupr h_mono H⟩
 else or.inl $ tendsto_at_top_at_top_of_monotone' h_mono H
+
+lemma tendsto_neg_nhds {α : Type*} [decidable_linear_ordered_add_comm_group α]
+  [no_bot_order α] [no_top_order α] [topological_space α]
+  [order_topology α] {a : α} :
+  tendsto has_neg.neg (nhds a) (nhds (-a)) :=
```

Comment:
I am surprised you need all these classes: having no bot and no top should not be relevant for this lemma. Maybe the current proof needs them (otherwise the linter would have complained), but it should be possible to remove them by changing the proof. Same thing in the next lemmas, where `densely_ordered` looks also suspicious